src/EFCore.PG.NTS/NetTopologySuiteMethodCallTranslator.cs

```diff
@@ -37,7 +37,7 @@ public virtual Expression Translate(MethodCallExpression e)
                 return new SqlFunctionExpression("ST_Difference",    typeof(Geometry), new[] { e.Object, e.Arguments[0] });
             case "Distance":
                 return new SqlFunctionExpression("ST_Distance",      typeof(double),   new[] { e.Object, e.Arguments[0] });
-            case "EqualsExact":
+            case "EqualsExact" when e.Object != null && e.Arguments[0] != null:
                 return Expression.Equal(e.Object, e.Arguments[0]);
```

Comment:
What is this for?

Reply:
`Expression.Equal(...)` throws if passed `null` for either argument. That's unlikely from user code directly, but could occur if passed a malformed expression from a previous visitor/translator.

Reply:
Thanks for the explanation!